### PR TITLE
Implement inotify file close events

### PR DESCRIPTION
**Pull request by lukassup**
INotify-specific events for IN_WRITE_CLOSE and IN_NOWRITE_CLOSE

### CODE REVIEW

**Comment by BoboTiG** [comment]
Hello @lukassup,

Thanks a lot for the patch!

Do you mind adding a line in https://github.com/gorakhargosh/watchdog/blob/master/changelog.rst and a test? It would be perfect :)

**Comment by BoboTiG** [comment]
Should we make a difference between "write" and "nowrite" events? The later should not trigger a `DirModifiedEvent` maybe?

File: changelog.rst [hunk 1]
```diff
@@ -3,10 +3,17 @@
 Changelog
 ---------
 
+0.10.5
+~~~~~~
+
+2020-xx-xx • `full history <https://github.com/gorakhargosh/watchdog/compare/v0.10.4...master>`__
+
+- [inotify] Add events for IN_WRITE_CLOSE and IN_NOWRITE_CLOSE (`#690 <https://github.com/gorakhargosh/watchdog/pull/690>`)
```
Comment (BoboTiG) [inline changelog.rst:11]: In fact, this is going into v0.10.4, no need for a new version :)

Comment (BoboTiG) [reply changelog.rst:11]: Also add youself in the line (in 0.10.4) "Thanks to our beloved contributors: ...".

**Review by BoboTiG — CHANGES_REQUESTED** [review]

**Comment by lukassup** [comment]
Thanks! Will do more extensive testing a bit later.

**Comment by BoboTiG** [comment]
Thanks to you 👍 
Let me know when you are ready for another review.

**Comment by BoboTiG** [comment]
Superseded by #747. Thanks a lot @lukassup for the initial patch (you've been credited correctly) 💪

**Pull request closed without merging**